.github/workflows/e2e-tests.yaml

```diff
@@ -16,7 +16,6 @@ jobs:
         node-version: 16.x
 
     - run: npm install
-    - run: npm run build
```

Comment:
It would be better to run it explicitly rather than making it a part of test scripts. Please keep this step here and remove it from the `pretest` script.